django_pony_express/services/base.py

```diff
@@ -245,16 +258,8 @@ def _build_mail_object(self) -> EmailMultiAlternatives:
         mail_attributes = self.get_context_data()
 
         # Render HTML body content
-        html_content = render_to_string(self.template_name, mail_attributes)
-
-        # Render TXT body part if a template is explicitly set, otherwise convert HTML template to plain text
-        if not self.template_txt_name:
-            h = html2text.HTML2Text()
-            # Set body width to "infinite" to avoid weird line breaks
-            h.body_width = 0
-            text_content = h.handle(html_content)
-        else:
-            text_content = render_to_string(self.template_txt_name, mail_attributes)
+        html_content = self.generate_html_content(mail_attributes)
```

Comment:
issue: I think it would be great to have two unittests for these new methods.

Reply:
I will try to write these this evening.

Reply:
❤️

Reply:
added tests.